CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [Unreleased](https://github.com/voxpupuli/puppet-php/tree/HEAD)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-php/compare/v8.0.3...HEAD)
```

Comment:
That doesn't seem right. Re-generate the CHANGELOG

Reply:
Looks like doing all the stuff in one line does not work: `bundle exec rake module:bump:minor strings:generate:reference changelog`